repos/fdbt-site/src/pages/products/productDetails.tsx

```diff
@@ -366,6 +367,17 @@ const createProductDetails = async (
         productDetailsElements.push({ id: 'time-restriction', name: 'Only valid during term time', content: ['Yes'] });
     }
 
+    if ('cap' in ticket && ticket.cap) {
+        const cap = (await getCapByNocAndId(noc, ticket.cap.id)) as Cap;
+
+        productDetailsElements.push({
+            id: 'cap',
+            name: 'Cap',
+            content: [`${sentenceCaseString(cap.capDetails.name)} - £${cap.capDetails.price}`],
```

Comment:
If we have created caps then it is possible we didn't assign any so we should have and N/A option

Reply:
Agree with Alaina